template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/LoginWith2fa.cshtml.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Company.WebApplication1.Data;
+
+namespace Company.WebApplication1.Pages.Account
+{
+    public class LoginWith2faModel : PageModel
```

Comment:
Class names beginning with numbers are the work of the devil.

It's in keeping with the page name, which seems to be the standard here.